JesusFilm/core
# fix: eng 1091 upload video error

# Description

### Issue

If an upload video error occurs but the user tries again, there error text still remains as well as the uploading/processing text

[Link to Linear Todo](https://linear.app/jesus-film-project/issue/ENG-1091/upload-video-error)

### Solution

Set error state to false onDrop

# External Changes

Refactored test class

# How to test
When a video upload error occurs (maybe forced by network change eta), the user should be able to retry again.
On another attempt you should never have both the old error text and the new uploading/processing text together.
